staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go

```diff
@@ -99,7 +99,13 @@ func (c *AggregationController) processNextWorkItem() bool {
 		return false
 	}
 
-	klog.Infof("OpenAPI AggregationController: Processing item %s", key)
+	if aggregator.IsLocalAPIService(key.(string)) {
+		// for local delegation targets that are aggregated once per second, log at
+		// higher level to avoid flooding the log
+		klog.V(5).Infof("OpenAPI AggregationController: Processing item %s", key)
+	} else {
+		klog.Infof("OpenAPI AggregationController: Processing item %s", key)
```

Comment:
in failure cases, how often is this logged?

Reply:
the key is re-queued by a rate limiter with 1 min base delay and 1 hr max delay (backoff exponentially)